todo/src/components/Todo.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { connect } from "react-redux";
+
+import { addTask, toggleDone } from "../actions";
+
+class ToDoList extends React.Component {
+    state = {
+        newTodo: ""
+    };
+    handleChanges = e => {
+        this.setState({ newTodo: e.target.value});
+    };
+    addTask = e => {
+        e.preventDefault();
+        this.props.addTask(this.state.addTask);
+    };
+    toggleDone = (e, index) => {
+        this.props.toggleDone(index);
+    };
+    render(){
+        return (
+            <React.Fragment>
+            <div className = "todo-list">
+                {this.props.tasks.map((task, index)=> (
+                    <h4 onClick={e => this.toggleDone(e, index)} key={index}>
+                        {task.name}
+                        {task.done && <i className="fas fa-dragon" />}
+                    </h4>
+                ))}
+            </div>
+            <input 
+                type="text"
+                value={this.state.newTodo}
+                placeholder="theres more things to do"
+                onChange={this.handleChanges}
+            />
+            <button onClick={this.addTask}>add task</button>
```

Comment:
encapsulate the input and the button in a form if you want to be able to hit enter upon entering a todo. Otherwise you will have to click and click only. No keyboard function.